test/NuGet.Core.Tests/NuGet.Protocol.Tests/Plugins/Logging/PluginLoggerTests.cs

```diff
@@ -84,13 +87,18 @@ public void Write_WhenLoggingIsEnabled_WritesToStringResult()
                 var environmentVariableReader = CreateEnvironmentVariableReaderMock(
                     isLoggingEnabled: true,
                     testDirectory: testDirectory);
-                var logMessage = new RandomLogMessage();
-
+                RandomLogMessageWithTime logMessage;
+                DateTimeOffset loggerInit;
+                DateTimeOffset randomMessage;
```

Comment:
How about `loggerInitLoggedAt` and `randomMessageLoggedAt` to suggest that these variables represent points in time?  Otherwise, they read like something entirely different (`randomMessage` == a random message) and makes reading [this line](https://github.com/NuGet/NuGet.Client/pull/3096/files#diff-d1ebcbddfb4f9047d15fee1b6debf550R119) confusing (how do you add a second to a message?).